server/Util.js

```diff
@@ -80,15 +81,15 @@ module.exports = {
     });
   },
 
-  runPredict: function (csv_file_name, job_id, timer, target_name, email, job_name, callback = "") {
+  runPredict: function (csv_file_name, job_id, timer, target_name, user_email, job_name, callback = "") {
```

Comment:
This change makes it so that users don't get sent bbml emails, they get sent our custom email on the callback, and instead the bbml emails get sent to the eautoml email